_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,269 @@
+
+**Note:** [Subclass roles](https://www.w3.org/TR/wai-aria-1.1/#subclassroles) of [required owned elements][] are not automatically included as possible [required owned elements][]. For example, the `treeitem` role is not a [required owned elements][] for [`list`](https://www.w3.org/TR/wai-aria-1.1/#list), even though `treeitem` is a [subclass role](https://www.w3.org/TR/wai-aria-1.1/#subclassroles) of `listitem`.
+
+## Assumptions
+
+If the [explicit semantic role][] on the target element is incorrectly used, and any relationships between elements are already programmatically determinable, failing this rule may not result in accessibility issues for users of assistive technologies, and it should then not be considered a failure under [WCAG success criterion 1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships).
+
+## Accessibility Support
+
+- User agents do not all have the same accessibility tree. Particularly the method of deriving which element owns which other elements varies between browsers. This can lead to different results for this rule, depending on which accessibility tree is used as input.
+- `aria-owns` has limited support in some user agents.
+- Assistive technologies are not consistent in how they handle situations where a [required owned element][] has a missing or incorrect role. This can lead to situations where inaccurate owned elements behave as expected in one assistive technology, but not in another.
+
+## Background
+
+- [Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
+- [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This element with the `list` role only owns elements with the `listitem` role. The `listitem` role is one of the [required owned elements][] for `list`.
+
+```html
+<div role="list">
+	<span role="listitem">Item 1</span>
+	<span role="listitem">Item 2</span>
+</div>
+```
+
+#### Passed Example 2
+
+This element with the `tablist` role only owns elements with the `tab` role. The `tab` role is one of the [required owned elements][] for `tablist`.
+
+```html
+<div role="tablist">
+	<span role="tab">Tab 1</span>
+	<span role="tab">Tab 2</span>
+</div>
+```
+
+#### Passed Example 3
+
+This element with the `grid` role only owns elements with the `row` role, and the element with the `row` role only owns elements with the `cell` role. The `row` role is one of the [required owned elements][] for `grid`, and `cell` is one of the [required owned elements][] for `row`.
+
+```html
+<table role="grid">
+	<tr role="row">
+		<span role="cell">Item 1</span>
+	</tr>
+</table>
+```
```

Comment:
I think that Passed Examples 1, 2, and 3 are very similar (simple good stuff) with 3 being a bit different (nested roles with required ownership).
I suggest we remove either Passed Example 1 or Passed Example 2.